utils/src/main/java/io/neow3j/constants/NeoConstants.java

```diff
@@ -113,7 +113,7 @@
 
     //endregion
 
-    //region Transactions
+    //region Transactions & Contracts
```

Comment:
Maybe I'm missing something (or my memory needs to be refreshed a bit), but why are we using these "region" annotations?

Reply:
Those are just there for giving a bit of structure. Intellij recognizes those and allow for collapsing.